src/main/java/org/example/bookweb/dto/shopping/cart/AddCartItemDto.java

```diff
@@ -0,0 +1,13 @@
+package org.example.bookweb.dto.shopping.cart;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class AddCartItemDto {
```

Comment:
```suggestion
public class CartItemRequestDto {
```
You can use java record for DTO